### DISCUSSION TITLE
DB table soundentries_dbc - Why Field are different?

### DISCUSSION THREAD

**Discussion by The-GhostRider** [azerothcore/azerothcore-wotlk#4503] [General]
why soundentries.dbc has field with a different name than the DB table soundentries_dbc?

soundentries.dbc
```
ID
field1
field2
field3
field4
field5
field6
field7
field8
field9
field10
field11
field12
field13
field14
field15
field16
field17
field18
field19
field20
field21
field22
field23
field24
field25
field26
field27
field28
field29
```
soundentries_dbc
```
ID
SoundType
Name
File_1
File_2
File_3
File_4
File_5
File_6
File_7
File_8
File_9
File_10
Freq_1
Freq_2
Freq_3
Freq_4
Freq_5
Freq_6
Freq_7
Freq_8
Freq_9
Freq_10
DirectoryBase
Volumefloat
Flags
MinDistance
DistanceCutoff
EAXDef
SoundEntriesAdvancedID
```

**Comment by Kitzunu** [comment]
Because people can figure out what the fields do, and therefore give them a descriptive name instead of "Field8"

**Reply by The-GhostRider** [reply]
> Because people can figure out what the fields do, and therefore give them a descriptive name instead of "Field8"

ok, tnks, I imagined it, but I wanted confirmation, it was just for my knowledge of it.

**Reply by The-GhostRider** [reply]
@Kitzunu btw that table is empty..

**Reply by Kitzunu** [reply]
yea it is supposed to be like that

**Comment by FrancescoBorzi** [comment]
@Yehonal @Winfidonarleyan
